lib/carin_for_blue_button_test_kit/client/v2.0.0/claim_data_request_tests/eob_outpatient_claims_data_request_test.rb

```diff
@@ -7,10 +7,12 @@ class C4BBClientEOBOutpatientSubmitClaimsDataRequestTest < Inferno::Test
     include CarinForBlueButtonTestKit::ClientValidationTest
 
     id :eob_outpatient_claims_data_request_test
-    title 'Outpatient Institutional ExplanationOfBenefit resources related to Carin claims data are gathered'
+    title 'Outpatient Institutional ExplanationOfBenefit resources related to CARIN claims data are gathered'
     description %(
-      This test will look through all returned Outpatient Institutional ExplanationOfBenefit resources for a specific
-      expected Carin ExplanationOfBenefit resource on the Inferno Reference Server.
+      This test will look through all returned ExplanationOfBenefit resources for a specific expected
+      ExplanationOfBenefit resource that supports the
+      CARIN for Blue Button [Outpatient Institutional ExplanationOfBenefit profile](https://hl7.org/fhir/us/carin-bb/STU2/StructureDefinition-C4BB-ExplanationOfBenefit-Outpatient-Institutional.html)
+      on the Inferno Reference Server.
```

Comment:
On this and other tests, need to get rid of the Inferno Reference Server detail.  I'm trying to decide how specific to be about our testing procedure here and I think we can be less specific.  I think we should say

>This test verifies that an instance returned by requests made by the client is an ExplanationOfBenefit resource that conforms to the CARIN for Blue Button [Outpatient Institutional ExplanationOfBenefit profile](https://hl7.org/fhir/us/carin-bb/STU2/StructureDefinition-C4BB-ExplanationOfBenefit-Outpatient-Institutional.html).

That keeps the implementation detail around looking for a specific detail out of the description.